extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -90,23 +91,32 @@ iframe.swg-dialog {
   font-weight: 400 !important;
 }
 
-/* TODO(#38112): confirm with UX about if there's better animation to be used. */
+[dir="rtl"] .i-amphtml-story-subscriptions-dialog-banner-button {
+  transform: translate(0.2em) !important;
+}
+
 .i-amphtml-story-subscriptions-dialog-banner-button.i-amphtml-story-subscriptions-dialog-banner-button-visible {
   opacity: 1 !important;
+  transform: translate(0) !important;
 }
 
 .i-amphtml-story-subscriptions-dialog-banner-button:after {
   content: '' !important;
   width: 0.93em !important;
   height: 0.93em !important;
   margin: 0 0.6em !important;
+  transform: translate(-0.1em) !important;
```

Comment:
This needs to have a transition or else it will not animate.

A dev tools tip for toggling classes to debug animations:
https://user-images.githubusercontent.com/3860311/165312969-7f9d2d14-456c-4e5b-a13e-a159b6a3e80a.mp4

Reply:
This is super useful, thanks! I now use the same value for scale and different values for translate so that only the translate would be animated.